src/storage/mvcc/txn.rs

```diff
@@ -697,10 +697,24 @@ impl<S: Snapshot> MvccTxn<S> {
                         }
                     }
                     None => {
-                        // Returning 0, 0 will allow the client to clean secondary locks. Put a
```

Comment:
Maybe still keep this sentence of comment?

Reply:
I think the new comments already says what this sentence wants to say. By the way I found that cleanup and check_txn_status looks very similar. I  think I'll try to simplify them in another PR.